website/src/components/Schema/index.js

```diff
@@ -167,7 +167,7 @@ const Schema = ({ schema, menuItem, isLatestSchema }) => {
         return (
           <TagContainer>
             {isRestrictedField && <Tag type={TAG_TYPES.required} />}
-            {isDependentField && <Tag type={TAG_TYPES.dependency} />}
+            {isDependentField && <Tag type={TAG_TYPES.conditional} />}
```

Comment:
can we change the var name to `isConditionalField` for consistency?

Reply:
Yes! good call.